### DISCUSSION TITLE
Training on colab

### DISCUSSION THREAD

**Discussion by C27U** [TheLastBen/fast-stable-diffusion#306] [Q&A]
So I'm style training, and I'm doing 'resume training' method since i can't do it in one training because i have many images = higher training steps and colab can't run longer on free tiers. My question is since i'm training it on the same style, if i use the same identifier like (gwsz (1).jpg) and so on, what will happen (is there any cons or pros) if resume train the model on the same identifier? (Gwsz)

**Comment by TheLastBen** [comment]
When you want to continue the training, you keep everything as it is, same instance pictures with the same filenames.

**Reply by C27U** [reply]
But is there a cons if i use different identifier for the next 'resume training'?

**Reply by TheLastBen** [reply]
With a a different name you would be creating another instance unrelated to the first one.

**Reply by C27U** [reply]
Oh and I'm training the model continuously with different images but same identifier (artist style), will it be fine?

**Reply by TheLastBen** [reply]
If you train on different style with the same instance name, you will mix them all.

**Reply by C27U** [reply]
Thanks, got it.
